robottelo/rerun_failures/uncollector.py

```diff
@@ -0,0 +1,111 @@
+import logging
+
+import pytest
+
+from robottelo.config import settings
+from robottelo.report_portal.portal import ReportPortal
+
+LOGGER = logging.getLogger(__name__)
+
+
+def _get_launch(rp):
+    """Returns the latest launch of satellite"""
+    _version = settings.server.version
+    sat_version = f'{_version.base_version}.{_version.epoch}'
+    launch = next(iter(rp.launches(sat_version=sat_version).values()))
```

Comment:
I don't see any handling of the ValueError exception that will happen here if we don't match the version in `rp.launches` call.

Either need to return `None` here or continue to let the exception raise - but in either case handle it within the hook and return with an unmodified test collection.

Reply:
This has been handled in report portal module itself.